pep-0570.rst

```diff
@@ -267,10 +281,14 @@ can be called. It would restrict arguments to be called as positional-only,
 while not allowing them to be called as keyword arguments.
 
 Previously, PEP 457 proposed to define the syntax, but with a much broader
-scope. This PEP takes the original proposal a step further that by justifying
+scope. This PEP takes the original proposal a step further by justifying
 the syntax and providing an implementation for the ``/`` syntax in function
 definitions.
 
+-----------
```

Comment:
Wow — the subsections give more clarity within the rationale. 

Indicating these subsections explicitly was something I was struggling to define when I was working on #972.

👍